src/application/migrations/011_remove_prefix_from_fkey_constraints.php

```diff
@@ -68,6 +68,10 @@ public function up()
             ADD CONSTRAINT `user_settings_users` FOREIGN KEY (`id_users`) REFERENCES `ea_users` (`id`)
             ON DELETE CASCADE
             ON UPDATE CASCADE');
+                                
+        // The name of these indexes changed in the file structure.sql of EA! 1.3.1 so we need to update it
+        $this->db->query('ALTER TABLE ea_secretaries_providers DROP KEY fk_ea_secretaries_providers_1, ADD KEY id_users_secretary (id_users_secretary)');
```

Comment:
Some lines above you we dropped the foreign key already. Do you thing this is really necessary?

<small>
<table>
<tr>
<td><img src="http://easyappointments.org/img/easyappointments-signature.png"></td>
<td>&nbsp;&nbsp;Alex Tselegidis, Easy!Appointments Creator <br>
&nbsp;&nbsp;<a href="http://alextselegidis.com">Need a customization? Contact me in person!</a></td>
</tr>
</table>
</small>

Reply:
Here we didn't drop foreign keys but two indexes which had ambiguous name and we renamed it. 
You noticed it and changed it in `structure.sql` in the commit d23ec0483cde47010869f0445d91b7104ed3079a so we applied this change to migration process. 
But this is not necessary since it does not cause issues.